versioned_sidebars/version-1.3-sidebars.json

```diff
@@ -215,71 +215,71 @@
         {
           "type": "link",
           "label": "What is Optimize?",
-          "href": "/optimize/3.7.0/components/what-is-optimize/"
+          "href": "/optimize/components/what-is-optimize/"
```

Comment:
Similar to the other sidebars file -- removing the version from the paths, everywhere that the sidebars cross docs instances.